RELEASE-NOTES.md

```diff
@@ -4,6 +4,61 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+-
+
+### Components:
+- **ContextSelector:** Updated signature for context selector ([#3697](https://github.com/patternfly/patternfly-react/pull/3697))
```

Comment:
WE have been going for sentence case with the names. So this should be "Context selector".  Same applies for the other components below

Reply:
Thank you!